packages/service-library/tests/rabbitmq/test_rabbitmq_rpc_router.py

```diff
@@ -0,0 +1,94 @@
+# pylint:disable=redefined-outer-name
+# pylint:disable=unused-argument
+
+from collections.abc import Awaitable, Callable
+
+import orjson
+import pytest
+from faker import Faker
+from servicelib.rabbitmq import RabbitMQClient
+from servicelib.rabbitmq_rpc_router import RPCRouter
+from servicelib.rabbitmq_utils import RPCMethodName, RPCNamespace
+
+pytest_simcore_core_services_selection = [
+    "rabbit",
+]
+
+
+@pytest.fixture
+def rabbitmq_rpc_client(
+    rabbitmq_client: Callable[[str], RabbitMQClient]
+) -> Callable[[str], Awaitable[RabbitMQClient]]:
+    async def _creator(name: str) -> RabbitMQClient:
+        rpc_client = rabbitmq_client(name)
+        assert rpc_client
+        await rpc_client.rpc_initialize()
```

Comment:
@GitHK @sanderegg This is a good example for a question i want to ask. here i want a rpc client (see? you even write that in the fixture!)

 Then Why don't we have a client specific for rpc? 
Can you tell me a use-case in which a `rabbitmq_client` uses both rpc and pub/sub interfaces at the same time??

Reply:
I agree. I will have a look now if this is easily doable

Reply:
but in a separate PR

Reply:
as for the use-case. I think it does not exist

Reply:
I think the RPC is an extension on top of rabbitmq. What will happen when we have such a use case? Will I have different clients with different connections to rabbitmq? Is that what we want anyway?

Reply:
yes